src/diffusers/loaders.py

```diff
@@ -946,11 +946,15 @@ def _modify_text_encoder(self, attn_processors: Dict[str, LoRAAttnProcessor]):
                 lora_layer = getattr(attn_processors[name], self._get_lora_layer_attribute(name))
                 old_forward = module.forward
 
-                def new_forward(x):
-                    return old_forward(x) + lora_layer(x)
+                # create a new scope that locks in the old_forward, lora_layer value for each new_forward function
```

Comment:
Here I'd not actually mind referring users to read this issue comment you made:
https://github.com/huggingface/diffusers/pull/3490#issuecomment-1555059060